iso15118/shared/security.py

```diff
@@ -1162,25 +1175,54 @@ class KeyPath(str, Enum):
     """
 
     # Mobility operator (MO)
-    CONTRACT_LEAF_PEM = PKI_PATH + "iso15118_2/private_keys/contractLeaf.key"
-    MO_SUB_CA2_PEM = PKI_PATH + "iso15118_2/private_keys/moSubCA2.key"
-    MO_SUB_CA1_PEM = PKI_PATH + "iso15118_2/private_keys/moSubCA1.key"
-    MO_ROOT_PEM = PKI_PATH + "iso15118_2/private_keys/moRootCA.key"
+    CONTRACT_LEAF_PEM = os.path.join(
+        PKI_PATH, "iso15118_2/private_keys/contractLeaf" ".key"
+    )
+    MO_SUB_CA2_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/moSubCA2.key")
+    MO_SUB_CA1_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/moSubCA1.key")
+    MO_ROOT_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/moRootCA.key")
 
     # Charge point operator (CPO)
-    SECC_LEAF_PEM = PKI_PATH + "iso15118_2/private_keys/seccLeaf.key"
-    CPO_SUB_CA2_PEM = PKI_PATH + "iso15118_2/private_keys/cpoSubCA2.key"
-    CPO_SUB_CA1_PEM = PKI_PATH + "iso15118_2/private_keys/cpoSubCA1.key"
-    V2G_ROOT_PEM = PKI_PATH + "iso15118_2/private_keys/v2gRootCA.key"
+    SECC_LEAF_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/seccLeaf.key")
+    CPO_SUB_CA2_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/cpoSubCA2.key")
+    CPO_SUB_CA1_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/cpoSubCA1.key")
+    V2G_ROOT_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/v2gRootCA.key")
 
     # Certificate provisioning service (CPS)
-    CPS_LEAF_PEM = PKI_PATH + "iso15118_2/private_keys/cpsLeaf.key"
-    CPS_SUB_CA2_PEM = PKI_PATH + "iso15118_2/private_keys/cpsSubCA2.key"
-    CPS_SUB_CA1_PEM = PKI_PATH + "iso15118_2/private_keys/cpsSubCA1.key"
+    CPS_LEAF_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/cpsLeaf.key")
+    CPS_SUB_CA2_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/cpsSubCA2.key")
+    CPS_SUB_CA1_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/cpsSubCA1.key")
     # The root is the V2G_ROOT
 
     # EV manufacturer (OEM)
-    OEM_LEAF_PEM = PKI_PATH + "iso15118_2/private_keys/oemLeaf.key"
-    OEM_SUB_CA2_PEM = PKI_PATH + "iso15118_2/private_keys/oemSubCA2.key"
-    OEM_SUB_CA1_PEM = PKI_PATH + "iso15118_2/private_keys/oemSubCA1.key"
-    OEM_ROOT_PEM = PKI_PATH + "iso15118_2/private_keys/oemRootCA.key"
+    OEM_LEAF_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/oemLeaf.key")
+    OEM_SUB_CA2_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/oemSubCA2.key")
+    OEM_SUB_CA1_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/oemSubCA1.key")
+    OEM_ROOT_PEM = os.path.join(PKI_PATH, "iso15118_2/private_keys/oemRootCA.key")
+
+
+class KeyPasswordPath(str, Enum):
+    """
+    Provides the path to private key passwords used for Plug & Charge. The encoding
+    format is indicated by the latter part of the enum name (_DER or _PEM)
```

Comment:
the docstring is incorrect. the DER, PEM stuff is not part of this ;)

Reply:
Oops...missed while copy pasting and editing. Have changed now.